libecole/include/ecole/observation/strongbranchingscores.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <memory>
+
+#include <nonstd/optional.hpp>
+#include <xtensor/xtensor.hpp>
+#include <xtensor/xview.hpp>
+
+#include "ecole/environment/state.hpp"
+#include "ecole/observation/abstract.hpp"
+
+namespace ecole {
+namespace observation {
+
+class StrongBranchingScoresObs {
+public:
+	xt::xtensor<double, 1> scores;
+};
+
+class StrongBranchingScores :
+	public ObservationFunction<nonstd::optional<StrongBranchingScoresObs>> {
+public:
+	bool pseudo_cands;
+	using Observation = nonstd::optional<StrongBranchingScoresObs>;
+	using Base = ObservationFunction<Observation>;
```

Comment:
Note that these aliases are no longer needed in the rest of the library, so feel free to remove them if you are not needing them.